buf.work.yaml

```diff
@@ -0,0 +1,10 @@
+version: v1
```

Comment:
These configs generated with `buf beta migrate-v1beta1` as described in https://docs.buf.build/configuration/v1beta1-migration-guide